algosdk/v2client/algod.py

```diff
@@ -307,13 +307,13 @@ def status_after_block(
         return self.algod_request("GET", req, **kwargs)
 
     def send_transaction(
-        self, txn: "transaction.Transaction", **kwargs: Any
+        self, txn: "transaction.GenericSignedTransaction", **kwargs: Any
```

Comment:
nit: do we still need the quotes?

Reply:
It failed CI the first time with a message about partially initialized module transaction, didnt dig deep but quoting fixed it